src/tools/genpolicy/README.md

```diff
@@ -18,32 +18,12 @@ The Policy auto-generated by `genpolicy` is typically used for implementing conf
 
 # Building `genpolicy` from source code
 
-## Install build dependencies
+Build in docker container:
 
-Example for Ubuntu 22.04.3:
-
-```bash
-$ sudo apt-get update
-$ sudo apt-get install -y build-essential cmake curl git musl-dev musl-tools
-$ curl --proto '=https' --tlsv1.3 https://sh.rustup.rs -sSf | sh
-$ source "$HOME/.cargo/env"
-$ arch=$(uname -m)
-$ rustup target add "${arch}-unknown-linux-musl"
-```
-
-# Build `genpolicy`
-
-```bash
+```sh
 $ git clone https://github.com/kata-containers/kata-containers.git
-$ cd kata-containers/src/tools/genpolicy
-$ source "$HOME/.cargo/env"
-$ make && make install
-```
-
-If you want to use `LIBC=gnu` instead of the default `LIBC=musl`, change the last step above to:
-
-```bash
-$ LIBC=gnu make && LIBC=gnu make install
+$ cd kata-containers
+$ tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh --build=genpolicy
```

Comment:
Can you also add a new target to `/tools/packaging/kata-deploy/local-build/Makefile`? That Makefile drives building assets, I don't believe we call `kata-deploy-binaries.sh` directly in general.

Reply:
And we'll probably also want to add genpolicy to `/.github/workflows/build-kata-static-tarball-amd64.yaml`.

Reply:
Done, thanks guys!